types/mongorito/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
Updated, and linter issues fixed